server.js

```diff
@@ -99,6 +99,14 @@ passport.use(new Strategy(
 const ImageServer = new  imgSteam.http.Connect(imageSteamConfig);
 const imageHandler = ImageServer.getHandler();
 
+/**
+ * Most errors is not found
+ * @TODO: requires debugging if other errors are handled by server
+ */
+ImageServer.on('error', (err) => {
+  // Don't log 404 errors, so we do nothing here.
+});
+
```

Comment:
Why do we need this if this is empty?

Reply:
Yes, but I don't know why. The server gives not the correct http error when I remove this listener.

Reply:
It has a TODO above it, so I think we can push this for now.